src/test/java/com/waverleysoftware/LoginToApplication.java

```diff
@@ -0,0 +1,32 @@
+package com.waverleysoftware;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class LoginToApplication extends BaseClass {
+    WebElement username = driver.findElement(By.id("email"));
+    WebElement password = driver.findElement(By.id("pass"));
```

Comment:
pure `WebElement` fields are used with `@FindBy` annotation and `PageFactory`... anyway, `By` class is more preferable for storing locators...